index.bs

```diff
@@ -501,31 +501,31 @@ dictionary StreamPipeOptions {
 Instances of {{ReadableStream}} are created with the internal slots described in the following
 table:
 
-<table>
+<table dfn-for="ReadableStream">
  <thead>
   <tr>
    <th>Internal Slot
    <th>Description (<em>non-normative</em>)
  <tbody>
   <tr>
-   <td>\[[disturbed]]
+   <td><dfn id="rs-slot-disturbed" lt="[[disturbed]]">\[[disturbed]]</dfn>
```

Comment:
The fact that `lt=""` is required here seems like a Bikeshed bug (maybe it's not doing the escaping in the right order). Maybe file an issue on https://github.com/tabatkins/bikeshed?

Reply:
Looks like it has already been fixed in Bikeshed, not sure when exactly. I'll remove the `lt=""` again. 🙂